test/views/CalculationViewSpec.scala

```diff
@@ -300,5 +296,77 @@ class CalculationViewSpec extends SpecBase with ViewBehaviours with ViewUtils {
 
       behave like pageWithExpectedMessages(expectedContent)(docWithOnlyOneCalculation)
     }
+
+    "it is an additional penalty and the user is an Agent (no estimate)" must {
```

Comment:
"it is an additional penalty and the user is an Agent (no estimate)" can be "it is second payment penalty and the user is an Agent (no estimate)"